amaranth-chrome-ext/src/content.js

```diff
@@ -0,0 +1,18 @@
+const mlModelPath = chrome.runtime.getURL('assets/model/model.json');
+const tokenizerPath = chrome.runtime.getURL('assets/tokenizer.json');
+
+async function main() {
+  // Load tokenizer dictionary
+  const response = await fetch(tokenizerPath);
+  const tokenizerObj = await response.json();
+  const tokenizer = new Map(Object.entries(tokenizerObj));
+
+  console.log(tokenizer.entries());
```

Comment:
Do you want to log these to the console?

Reply:
I used these logs to make sure that the files were being read in correctly, but they probably don't need to be committed to the repo. I'll remove them!

Reply:
Fixed in https://github.com/googleinterns/amaranth/pull/39/commits/5de6bad3a00970d9334dbe8a414d031fb6b5d00c